modules/project-docs/pages/compatibility.adoc

```diff
@@ -7,7 +7,8 @@ Plus notes on Cloud, networks, and AWS Lambda.
 [abstract]
 {description}
 
-The Couchbase Ruby SDK is compatible with Ruby 2.5, 2.6, and 2.7.
+The Couchbase Ruby SDK is compatible with Ruby 2.6, 2.7, and 3.0.
+// We follow list of supported versions at https://www.ruby-lang.org/en/downloads/branches/
```

Comment:
Did you mean for this to be commented out?

Reply:
Yes, for future updates.